core/src/main/scala/com/phaller/rasync/HandlerPool.scala

```diff
@@ -444,4 +449,20 @@ class HandlerPool(parallelism: Int = 8, unhandledExceptionHandler: Throwable =>
 
   def reportFailure(t: Throwable): Unit =
     t.printStackTrace()
+
+  /**
+   * Interrupt the computation of cells. It can be resumed using the `resume` method.
+   */
+  def interrupt(): Unit = {
+    isInterrupted = true
+  }
+
+  /**
+   * Resume the computation if the execution was interrupted. Don't do anything if the execution was not interrupted.
+   */
+  def resume(): Unit = {
+    isInterrupted = false
+    interruptLatch.countDown()
+    interruptLatch = new CountDownLatch(1)
```

Comment:
Would it simplify finding bugs if this method would throw an exception (e.g., `IllegalStateException`) in case it is attempted to resume a non-interrupted pool?

Reply:
In the `PropertyStore` the interrupt is implemented as `@volatile var isInterrupted: () ⇒ Boolean`. Therefore I call `resume` every time the client waits for quiescense, which should resume the store if it was interrupted.